src/material/core/ripple/ripple-renderer.ts

```diff
@@ -296,6 +310,7 @@ export class RippleRenderer implements EventListenerObject {
 
   /** Destroys the given ripple by removing it from the DOM and updating its state. */
   private _destroyRipple(rippleRef: RippleRef) {
+    const rippleEventListeners: RippleEventListeners | null = this._activeRipples.get(rippleRef)!;
```

Comment:
Feels a little unsafe that we manually need to re-add the `null` type-possibility here. How about something like:

```suggestion
    const eventListeners =  this._activeRipples.get(rippleRef) ?? null;
```

Reply:
Makes sense.